dev/crypto.md

```diff
@@ -80,22 +82,41 @@ below specifies each prefix (in quotation marks):
        - "aS": A _Settlement_.
 
 
-## Hash Function
+## Hash Functions
 
+### SHA512-256
 Algorand uses the [SHA-512/256 algorithm][sha] as its primary
 cryptographic hash function.
 
 Algorand uses this hash function to (1) commit to data for signing and
 for the Byzantine Fault Tolerance protocol, and (2) rerandomize its
 random seed.
 
+### SUBSET-SUM
+Algorand uses [SUBSET-SUM algorithm][sumhash] which is a quantum-resilient hash function.
```

Comment:
I believe that SHA-512/256 is also conjectured to be quantum-resilient.
It may be a bit confusing to state that SUBSET-SUM is quantum-resilient therefore.

Reply:
Can you please provide the parameters?
Can you provide the estimated security? (e.g., 128 bits)
If the parameters are not the ones proposed in the paper, can you provide a security analysis?

Reply:
Can you provide the way the output is encoded (big endian, little endian, ...)?

Reply:
There is a link to the full spec which contains all the details. do you think we should add those parameters anyway?

Reply:
The document looks good.